src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/PackageUpdaterStep.cs

```diff
@@ -57,6 +58,7 @@ public class PackageUpdaterStep : UpgradeStep
             IOptions<PackageUpdaterOptions> updaterOptions,
             IPackageRestorer packageRestorer,
             IEnumerable<IDependencyAnalyzer> packageAnalyzers,
+            IPackageAnalyzer packageAnalyzer,
```

Comment:
The name of this should probably be broadened to be `IDependencyAnalyzerRunner` or something